package.json

```diff
@@ -17,14 +17,28 @@
     "executablePath": {
       "default": "tidy",
       "title": "Full path to the `tidy` executable",
+      "order": 1,
       "type": "string"
     },
+    "executableArguments": {
+      "default": [
+        "-utf8"
+      ],
+      "title": "Tidy Executable Arguments",
+      "description": "Additional arguments to pass to the Tidy executable when invoked.<br/><br/>The arguments specified here will be appended to arguments required for this linter to work.<br/><br/>Note that this option is an array. Each executable argument should be an element in the array.",
```

Comment:
Yeah, that's a better way to go. I was attempting to nudge a user of Atom's built-in `settings-view` to know what to input while also staying neutral about the method used to modify the option. In reality, most users will probably just use the default view and will want to know what to type there, and anyone else will probably know what to do with their method of choice.

Since this PR conflicted after merging #92, I've squished the branch down (including the most recent commit to address your feedback) and rebased it onto `master`.